recipes-private/nsls2ptycho/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set token = environ.get('GH_TOKEN') %}
```

Comment:
I like this trick! If I use my token, I guess it won't work as I don't have access to your repo, right? How can we make it more sustainable?

Reply:
If I add you to the repo, you'll have access. The only drawback I encounter for now is that if we forget to set the env variable, the error message is obscure :D  

For the long term, I am hoping that @stuartcampbell will allow both the frontend and the backend repos to be transferred to NSLS-II so that we can assign team privileges. For example, DAMA will have the write access, and collaborators will have read-only (or write if they want to contribute) access.

Reply:
Since it's a manual process of building anyway, it's OK. Having a broken environment in a container is safe, and the package won't be uploaded, so I'm not concerned much.

Could you please add me as a collaborator on that private repo before we move the repo(s) to the new home? Thanks.

Reply:
Done.